data/en/imagecaptcha.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "imageCaptcha",
+	"type": "function",
+	"syntax": "imageCaptcha()",
+	"returns": "any",
+	"related": ["imagewritetobrowser"],
+	"description": "Provides information about the time when the current application scope was created.",
+	"params": [
+            {"name":"text","description":"Text in the image captcha.","values":[],"default":"","type":"string","required":true},
+            {"name":"height","description":"Height of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"width","description":"Width of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"difficulty","description":"Width of the image captcha.","values":[],"default":"medium","type":"string","required":false},
```

Comment:
According to the Lucee doc, the default is `low`. Also, the values should be `low`, `medium`, `high`.

Reply:
Updated default to low.